R/form_schema_ext.R

```diff
@@ -221,38 +173,65 @@ form_schema_ext <- function(flatten = FALSE, odata = FALSE, parse = TRUE, pid =
 
       if (has_translation) {
         # finds all translations related to this path:
-        id <- sub("')", "", sub("jr:itext\\('", "", xml2::xml_attr(this_rawlabel, "ref")))
```

Comment:
Great to see prefixed function calls. For better or worse, I prefix all external functions, which makes debugging so much easier.

Reply:
I indeed prefer the long way.. it is easier to understand and follow